reboot_toolkit/toolkit_backend.py

```diff
@@ -584,7 +754,9 @@ def filter_pop_df(
     :return: the filtered and down-sampled population dataframe
     """
 
-    assert len(mean_df) == len(std_df), "Mismatched frames between mean and std population data"
+    assert len(mean_df) == len(

```

Comment:
We should replace the assert statement here with a regular conditional that raises an exception.